src/AppInstallerRepositoryCore/Rest/HttpClientHelper.h

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+#pragma once
+#include "pch.h"
+#include "cpprest/http_client.h"
+#include "cpprest/json.h"
+
+using namespace web;
+using namespace web::json;
+using namespace web::http;
+using namespace web::http::client;
+
+namespace AppInstaller::Repository::Rest
+{
+	struct HttpClientHelper
+	{
+		HttpClientHelper(utility::string_t url);
+
+		pplx::task<web::http::http_response> Post(const json::value& body);

```

Comment:
the http_client::request method called in MakeRequest is non-const, and since all the methods call MakeRequest, we cant make these methods const. Please correct me if wrong.